framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -8,7 +8,8 @@ import {
 } from 'chakra-react-select'
 import { PlusSolid } from '@northlight/icons'
 import { Box, Icon } from '@chakra-ui/react'
-import { any, isEmpty, toLower, trim } from 'ramda'
+import { any, compose, isEmpty, prop, toLower, trim } from 'ramda'
+import { uniqBy } from 'yafu'
```

Comment:
you can get uniqBy from ramda aswell, not that it is functionally different, but the bundle size would be smaller since we already have ramda but not yafu as dependency

Reply:
nvm on this comment, Oles enlightened me that it's not available in the version of ramda that we are using